docs/news/README.md

```diff
@@ -0,0 +1,9 @@
+# Unreleased Notes
+
+`NEWS.md` in the root directory of this project contains release notes for all
+previous releases. This directory contains a file for each _unreleased_ change
+that is already on `main`.
+
+These are single files in this directory to reduce the likelihood of a merge
+conflict. They will be deleted with their content moved into `NEWS.md` when
+doing a release.
```

Comment:
I think we could take this one step further, and have each news entry be a free-form full-length blog-post type thing. Then whomever makes the NEWS file update can summarize them there (rather than from commits), and we could move the entries into a per-release directory when we do the release, for posterity. We could also post the entries to our website as blog posts when that makes sense (maybe eventually always?).

Reply:
Genius! Changed it completely to do this instead of my first proposal. I really like this.